### DISCUSSION TITLE
求助：这网站这么吃配置吗

### DISCUSSION THREAD

**Discussion by Erdbronie** [share121/inter-knot#1089] [其他]
![屏幕截图 2024-08-10 223946](https://github.com/user-attachments/assets/5c8e6a03-a426-40d5-9456-e8f3753ce939)
rt，一开这个网页GPU0占用就几乎拉满，有什么解决办法吗

**Comment by GanlinSama** [comment]
只能说edge是这样的，我GPU占用率都直接拉满的

**Comment by Y-0930** [comment]
我的也是

**Comment by likeseele** [comment]
我这也是，占用70%

**Comment by DonQuixote084** [comment]
虽然说edge是这样的，但是楼主也有可能是被植入了病毒，快看看进程里有没有一个叫farry的，是在伪装你的浏览器调用你的摄像头监视你

**Reply by Y-0930** [reply]
fairy：目前，该用户已被封禁

**Comment by su2569** [comment]
6

**Comment by 1451374815** [comment]
占用高，但频率其实不高，功耗也不大，楼主无需担心

**Comment by Kamisato100** [comment]
我用Chrome内存和GPU都吃的很少，挺流畅的，感觉优化比前几天好了很多

**Comment by Buer-Nahida** [comment]
我这边占用很少，楼主换个浏览器试试？
![图片](https://github.com/user-attachments/assets/42c16d02-7e8c-47cf-9bbe-4c8539cd2985)
